src/components/details/MatchDetails.jsx

```diff
@@ -10,13 +10,16 @@ import { BootstrapDialogTitle } from './modal/BoostrapDialogTitle';
 import MatchDetailsContent from './MatchDetailsContent';
 import Confirmation from '../feedback/Confirmation';
 import { label, isUser } from '../../helpers/usedFunctions';
+import Map from '../Map';
+import { Grid } from '@mui/material';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/FeNixCrew/MiEntradita-frontend/pullRequest?prid=8421134)
```suggestion
import { Grid } from "@mui/material";
```